genie/load.py

```diff
@@ -222,7 +221,10 @@ def _reorder_new_dataset(
         The re-ordered new dataset
     """
     # Columns must be in the same order as the original data
-    new_dataset = new_dataset[orig_database_cols]
+    if "ROW_ID" in orig_database_cols and "ROW_VERSION" in orig_database_cols:
+        new_dataset = new_dataset[orig_database_cols.drop(["ROW_ID", "ROW_VERSION"])]
```

Comment:
I wonder why we are dropping these when we'd likely need them later when deleting rows?